api/nodes/views.py

```diff
@@ -2,15 +2,22 @@
 
 from modularodm import Q
 from rest_framework import generics, permissions as drf_permissions
-from rest_framework.exceptions import PermissionDenied, ValidationError
+from rest_framework.exceptions import PermissionDenied, ValidationError, NotFound
```

Comment:
Keeping imports sorted in some way (such as alphabetical) is nice... but not a big deal.